tests/integration_tests/document_loaders/test_dir_text_loader.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
```

Comment:
nice! This test can actually be moved into unit tests, so it can be picked up during CI tests.

We're in the process of making sure we can run all tests that don't involve reaching out to external systems as part of the regular CI test runner